### PR TITLE
[DFC-466] One Login Analytics Implementation

### PR DESCRIPTION
**Pull request by di-aholme**
### What changed

- Updated common express to version 6.2.1
- Installed OneLogin Analytics package
- Updated environment variables to handle new GA4 logic
- Updated config and build scripts to use the new analytics package

### Issue tracking

- [DFC-466](https://govukverify.atlassian.net/browse/DFC-466)

### Environment variables or secrets

- [x] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [x] Added to local startup repository

[DFC-466]: https://govukverify.atlassian.net/browse/DFC-466?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by SophiaDWokoma — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -8,9 +8,8 @@
...
-    "build-js": "yarn build-js:application && yarn build-js:analytics && yarn build-js:all",
-    "build-js:analytics": "mkdir -p dist/public/javascripts; uglifyjs node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/**/*.js node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/ua/legacy/cookies.js node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/*.js --beautify -o dist/public/javascripts/analytics.js",
-    "build-js:application": "mkdir -p dist/public/javascripts; uglifyjs src/assets/javascripts/application.js  node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/application.js --beautify -o dist/public/javascripts/application.js",
+    "build-js": "yarn build-js:analytics && yarn build-js:all",
+    "build-js:analytics": "mkdir -p dist/public/javascripts; uglifyjs node_modules/@govuk-one-login/one-login-analytics/lib/analytics.js node_modules/@govuk-one-login/di-ipv-cri-common-express/src/assets/javascript/analytics/ua/legacy/cookies.js --beautify -o dist/public/javascripts/analytics.js",
```
Comment (di-fabs) [inline package.json:12]: This command doesn't use the init.js (UA script) and it's mandatory if we want to init and run UA

**Review by di-fabs — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (di-aholme) [reply package.json]: Updated

**Review by Caitlin-cooling — APPROVED** [review]
Changes look good to me on the understanding that there will be another PR with the page load scripts 👍

File: src/app.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ const sessionConfig = {
   ...(SESSION_TABLE_NAME && { sessionStore: dynamoDBSessionStore }),
 };
 
-const helmetConfig = require("@govuk-one-login/di-ipv-cri-common-express/src/lib/helmet");
+const helmetConfig = commonExpress.lib.helmet;
```
Comment (merlinc) [inline src/app.js:59]: Helmet is still not being exported - https://github.com/govuk-one-login/ipv-cri-common-express/blob/main/src/lib/index.js

Do you need this line?

**Review by merlinc — CHANGES_REQUESTED** [review]
The rest of the PR seems to have been merged already and no longer appears after a rebase.

Is this PR still required?

**Comment by di-aholme** [comment]
Changes merged as part of https://github.com/govuk-one-login/ipv-cri-kbv-front/pull/890

**Pull request closed without merging**